go.mod

```diff
@@ -51,6 +51,7 @@ require (
 	golang.org/x/crypto v0.0.0-20211215153901-e495a2d5b3d3
```

Comment:
The following dependencies are unused and can be removed from `go.mod`:
* github.com/hashicorp/go-uuid
* github.com/jcmturner/gofork
* github.com/klauspost/compress
* gopkg.in/jcmturner/aescts.v1
* gopkg.in/jcmturner/dnsutils.v1
* gopkg.in/jcmturner/gokrb5.v7
* gopkg.in/jcmturner/rpc.v1

Reply:
These are required dependencies for this PR

Reply:
Yep
> Never mind, I'm seeing all of these deps in `go.mod` when regenerating it from scratch.